content/structure.md

```diff
@@ -145,7 +166,20 @@ import '../../ui/accounts/accounts-templates.js';
 // Below here are the route definitions
 ```
 
-On the server, we import various modules into our `server/main.js` to set up the server environment:
+We then import both of these files in `/imports/startup/client/index.js`:
```

Comment:
When referencing files in the app, let's not use a leading `/`, even if it is more correct.